src/modules/core/components/NetworkHealth/NetworkHealthContent/NetworkHealthContent.css

```diff
@@ -39,19 +39,23 @@
   align-self: flex-start;
 }
 
-.actionsContainer button {
-  padding: 0;
-}
-
 .closeButton {
   composes: button from '~styles/reset.css';
   stroke: var(--text);
-}
 
-.closeButton i {
-  padding: 0;
+  &, & i {
```

Comment:
Question... does this `&, & i {` translate to the following?

```css
.closeButton,
.closeButton i {
    ...
}
```

Reply:
Exactly. 

But since you've brought it up, I'll de-obfuscate it for better readability

Reply:
Up to you!

That can definitely be a useful feature though, so... good to know, thanks!